docs/authenticated_api/http.md

```diff
@@ -452,14 +452,15 @@ The field described above is for type 'trade' only and will be omitted if recs_p
 ###Call
 
 >```text
-money/new_deposit_address
+GENMKT/money/new_deposit_address
```

Comment:
This should be BTCEUR. Possibly it works with GENMKT because BTC is the fallback main currency.